crates/wit-component/tests/interfaces/diamond-disambiguate/shared1.wit.print

```diff
@@ -0,0 +1,5 @@
+interface shared {
+  type t1 = u8
+
```

Comment:
Note: don't need to fix this in this PR as it is a preexisting issue with the wit printer, but we should probably do a pass on the newlines it prints to make the output more consistent; perfect for a follow-up PR.